libr/include/r_util/r_str.h

```diff
@@ -21,6 +21,8 @@ typedef enum {
 
 typedef int (*RStrRangeCallback) (void *, int);
 
+typedef struct { ut8 ch; ut8 hx; } charset[255];
```

Comment:
this should be consistent with other types in r2. It should be named something like `RCharset` IMO

Reply:
Good idea. I should add RCharset in RCore.

Reply:
I think defining it here, in r_str.h, is ok, because you are using it also in the signature of `r_charset_encode_str`, so moving this typedef in RCore would not work.

Reply:
Then I could define RCharset in r_str.h but use it in RCore.

Reply:
@ret2libc What about my new commit?